src/Extension.Tests/LessonTests.cs

```diff
@@ -58,9 +58,9 @@ public void can_use_on_evaluation_complete_handler_to_skip_to_a_specific_challen
             var lesson = new Lesson();
             var challenges = lesson.AddBlankChallenges("1", "2", "3");
             lesson.ChallengeController.Commit();
-            challenges["1"].OnEvaluationComplete((challenge, lesson) =>
+            challenges["1"].OnCodeSubmitted((challenge, lesson) =>
```

Comment:
If the challenge has a parent lesson, then it's unnecessary to pass two parameters here.

Reply:
updated